api-schema/xdb.yaml

```diff
@@ -585,6 +610,33 @@ components:
         dbColumn:
           type: string
           description: the column name that can be used in the database query, see below for example   
+    LoadLocalAttributesRequest:
+      type: object
+      properties:
+        localAttributeLoadingType:
+          $ref: '#/components/schemas/LocalAttributeLoadingType'
+        partialLoadingKeys:
+          type: array
+          items:
+            type: string
+        lockingKeys:
+          type: array
+          items:
+            type: string
```

Comment:
so it would be like this:
```
LoadLocalAttributesRequest:
   keysToLoadNoLock: array of string -- the keys to load without lock
   keysToLoadWithLock: array of string -- the keys to load with lock
   lockType: TableReadLockingPolicy -- the locks applied for keysToLoadWithLock
```